airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -21,6 +21,11 @@ class SlackStream(HttpStream, ABC):
     url_base = "https://slack.com/api/"
     primary_key = "id"
     page_size = 100
+    channel_filter = []
```

Comment:
```suggestion
```
I don't think this class attribute is useful as you set this as instance attribute defaulting to `[]` in your `__init__`

Reply:
done